examples/generative/diffusion/train.py

```diff
@@ -282,6 +311,18 @@ def main(cfg: DictConfig) -> None:
 # ----------------------------------------------------------------------------
 
 if __name__ == "__main__":
-    main()
+    parser = argparse.ArgumentParser()
```

Comment:
You don't need to use argparse to get the config file, hydra already has support for this. Please check out the CorrDiff example to see how this is done: https://github.com/NVIDIA/modulus/blob/main/examples/generative/corrdiff/train.py#L48

Reply:
Thanks for sharing the reference from CorrDiff example. The code related to argparse has been removed. The command lines for model training and sampling have been updated in the README.md accordingly.